src/extension/background-script/actions/accounts/select.ts

```diff
@@ -27,6 +28,8 @@ const select = async (message: MessageAccountSelect) => {
     // save the current account id once the connector is loaded
     await state.getState().saveToStorage();
 
+    await notifyAccountSwitched();
```

Comment:
Rene has voted for `accountChanged` so maybe we should update all the usages - `notifyAccountChanged`, etc.

Reply:
resolved